src/@types/map.d.ts

```diff
@@ -0,0 +1,17 @@
+interface MapProps {
+    data: any;
+    dataLayer?: any;
```

Comment:
I think of dataLayer like an object that represents the styling of data

Reply:
Also the `any` here is temporary (and a total copout of using the type system lol. I'll need to spend just a bit more time looking at this object structure and replace it with the `any`)